OTE landing page (#961)
open-edge-platform/training_extensions: OTE_landing_page.md

> # OpenVINO™ Training Extensions {#ote_documentation}

Reviewer: `{#ote_documentation}` what is this for?

Author: it's sphinx tag to build the doc on our website

[![python](https://img.shields.io/badge/python-3.8%2B-green)]()
[![black](https://img.shields.io/badge/code%20style-black-000000.svg)]()
[![mypy](https://img.shields.io/badge/%20type_checker-mypy-%231674b1?style=flat)]()
[![openvino](https://img.shields.io/badge/openvino-2021.4-purple)]()

OpenVINO™ Training Extensions (OTE) provide a suite of advanced algorithms to train
Deep Learning models and convert them using the [OpenVINO™
toolkit](https://software.intel.com/en-us/openvino-toolkit) for optimized

> inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sources and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).

Reviewer: It allows you to export and convert
or
It allows exporting and converting

> inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sources and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).

Reviewer: ```suggestion
inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sourced and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).
```

## Detailed Workflow
![](training_extensions_framework.png)

1. To start working with OTE, prepare and annotate your dataset. For example, on CVAT.

2. OTE train the model, using training interface, and evaluate the model quality on your dataset, using evaluation and inference interfaces.

> Note: prepare a saparate dataset or split the dataset you have for more accurate quality evaluation.

Reviewer 2: separate

> Note: prepare the saparate dataset or split the dataset you have for the more accurate quality evaluation.

Reviewer: ```suggestion
Note: prepare a separate dataset or split the dataset you have for the more accurate quality evaluation.
```

> Note: prepare the saparate dataset or split the dataset you have for the more accurate quality evaluation.

Reviewer: ```suggestion
Note: prepare the saparate dataset or split the dataset you have for more accurate quality evaluation.
```
